### PR TITLE
corrected version homework week1

### PR DESCRIPTION
**Pull request by alquinex**

### CODE REVIEW

File: Week1/homework/app.js [hunk 1]
```diff
@@ -2,10 +2,120 @@
...
+      p.textContent = bookInfo.author;
+    }
+  }
+
+  const bookCovers = {
+    So_Long_A_Letter: './images/So_Long_A_Letter.jpeg',
+    Macbeth: './images/Macbeth.jpeg',
+    Julius_Caesar: './images/Julius_Caesar.jpeg',
+    Silas_Marner: './images/Silas_Marner.jpeg',
+    The_Gods_Are_Not_To_Blame: './images/The_Gods_Are _Not _To _Blame.jpeg',
```
Comment (remarcmij) [inline Week1/homework/app.js]: The HTML validator gives this error message:

```
Error: Bad value ./images/The_Gods_Are _Not _To _Blame.jpeg for attribute src on element img: Illegal character in path segment: space is not allowed.

From line 13, column 602; to line 13, column 655

Rotimi</p><img src="./images/The_Gods_Are _Not _To _Blame.jpeg"></li><
```

File: Week1/homework/app.js [hunk 2]
```diff
@@ -2,10 +2,120 @@
...
+    Installing_And_Configuring_Windows_Server_2012_R2:
+      './images/Installing_And_Configuring_Windows_Server_2012.jpg'
+  };
+
+  function displayBooks() {
+    for (const key of Object.keys(bookCovers)) {
+      const li = document.getElementById(key);
+      const img = document.createElement('img');
+      li.appendChild(img);
+      img.setAttribute('src', `${bookCovers[key]}`);
```
Comment (remarcmij) [inline Week1/homework/app.js]: HTML validation error (for all `` tags):

```
Error: An img element must have an alt attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.

From line 13, column 88; to line 13, column 129

ama Ba</p><img src="./images/So_Long_A_Letter.jpeg"></li><
```

File: Week1/homework/app.js [hunk 3]
```diff
@@ -2,10 +2,120 @@
...
+    }
+  }
+
+  function main() {
+    createList();
+    displayBooks();
+  }
+  window.onload = main;
```
Comment (remarcmij) [inline Week1/homework/app.js]: Hi Alusine, there are some HTML validation errors, but the JS part is fine after our 1-on-1 session.

File: Week1/homework/app.js (hunk 2 above)
Comment (alquinex) [reply Week1/homework/app.js]: Hi Jim,
All errors have been fixed and correction made. It should be fine now

**Pull request closed without merging**